### PR TITLE
add npm link documentation

### PR DESCRIPTION
**Pull request by harunurhan**
Today @zzacharo and I realized that this is needed 😄

### CODE REVIEW

**Comment by zzacharo** [comment]
:+1: Perfect :D

File: README.md [hunk 1]
```diff
@@ -85,6 +85,35 @@ npm start
...
+### Working with npm link
+
+```bash
+# in ng2-json-editor
+
+# do initial build into dist
+npm run build
+
+# copy package.json to dist
+npm run copy
```
Comment (jmartinm) [inline README.md]: What was this for?

Comment (harunurhan) [inline README.md]: this is for copying `LICENSE` and `package.json`

**Comment by zzacharo** [comment]
@jmartinm `package.json` is needed for linking inside `dist` folder. Otherwise, it will link from the nearest folder that includes one! @harunurhan right?

File: README.md (hunk 1 above)
Comment (jmartinm) [inline README.md]: And why is it needed to work with `npm link` ?

Comment (zzacharo) [inline README.md]: `package.json` is needed for linking inside dist folder. Otherwise, it will link from the nearest folder that includes one.

**Comment by harunurhan** [comment]
Yes, the process should be same as publishing, otherwise `npm` looks for a `package.json` in parent directories and links not only `dist`, but a lot of other stuff.

**Pull request merged**